account_reconcile_oca/models/account_bank_statement_line.py

```diff
@@ -983,45 +983,58 @@ def create(self, mvals):
             ]
         )
         for record in result:
-            res = models._apply_rules(record, record._retrieve_partner())
-            if not res:
-                continue
-            liquidity_lines, suspense_lines, other_lines = record._seek_for_lines()
-            data = []
-            for line in liquidity_lines:
-                reconcile_auxiliary_id, lines = record._get_reconcile_line(
-                    line,
-                    "liquidity",
-                    move=True,
-                )
-                data += lines
-            reconcile_auxiliary_id = 1
-            if res.get("status", "") == "write_off":
-                data = record._recompute_suspense_line(
-                    *record._reconcile_data_by_model(
-                        data, res["model"], reconcile_auxiliary_id
-                    ),
-                    self.manual_reference,
-                )
-            elif res.get("amls"):
-                amount = self.amount_currency or self.amount
-                for line in res.get("amls", []):
-                    reconcile_auxiliary_id, line_datas = record._get_reconcile_line(
-                        line, "other", is_counterpart=True, max_amount=amount, move=True
-                    )
-                    amount -= sum(line_data.get("amount") for line_data in line_datas)
-                    data += line_datas
-                data = record._recompute_suspense_line(
-                    data,
-                    reconcile_auxiliary_id,
-                    self.manual_reference,
+            if not record.is_reconciled:
+                record._enqueue_auto_reconcile(models)
+        return result
+
+    def _enqueue_auto_reconcile(self, models):
+        # Install module account_reconcile_oca_queue to run this method in a job queue
+        self._auto_reconcile(models)
```

Comment:
With https://github.com/OCA/queue/blob/18.0/queue_job/models/base.py#L145 we don't need the two functions. can you review? This way, everything will be cleaner 😉 

We used that on edi-framework 

https://github.com/OCA/edi-framework/pull/192/commits/a2d277eae5bde8dd2dc4feccb0b315a128c1ee56